docs/BpfSyscallCompatibility.md

```diff
@@ -0,0 +1,133 @@
+# `bpf()` syscall compatibility with Linux ABI
+
+There is a desire to have first-class eBPF support for Windows for the Go ecosystem.
+The Go community prefers for libraries that can be easily cross-compiled and distributed.
+However, calling into libbpf via Go's foreign function interface (CGo) goes against this spirit, as it requires distributing C headers, a C toolchain, and other dependencies.
+
+To address this, one of the most popular ways to interact with eBPF on Linux is through the ebpf-go library, which intentionally avoids CGo.
```

Comment:
nit: add a link to the ebpf-go library so that is is clear, which implementation is talked about in the following text.
```suggestion
To address this, one of the most popular ways to interact with eBPF on Linux is through the [ebpf-go library](https://ebpf-go.dev/), which intentionally avoids CGo.
```